Code/GraphMol/MarvinParse/testMrvToMol.cpp

```diff
@@ -1138,6 +1218,22 @@ M  END
       }
     }
 
+    if (testToRun == "" || testToRun == "scsrFileTests") {
+      std::list<ScsrMolTest> scsrFileTests{
+          ScsrMolTest(
+              "153944501_original_structure.mol", true, 859, 1025,
+              {66, 67, 68, 72, 73, 74} /* Some but not all hbonds in the mol */,
+              128),
+          ScsrMolTest("rnaTest.mol", true, 22, 24, {}, 4)};
+
+      for (auto &scsrFileTest : scsrFileTests) {
+        BOOST_LOG(rdInfoLog) << "Test: " << scsrFileTest.fileName << std::endl;
+
+        printf("Test\n\n %s\n\n", scsrFileTest.fileName.c_str());
```

Comment:
I know we have a bunch of these already, but I would love to not make the problem worse. Please don't output status information like this while the tests are running.

Ideally a successful test would generate no output beyond the number of tests run and the number passed/failed. In the case of the old-style tests (like this one), I'm also fine with outputting the names of the tests themselves (as you do at the top of `testMarvinFromScsr()`.

Reply:
I removed the printf statement